JitsiConference.js

```diff
@@ -595,13 +595,21 @@ JitsiConference.prototype.addTrack = function(track) {
 };
 
 /**
- * Fires TRACK_AUDIO_LEVEL_CHANGED change conference event.
+ * Fires TRACK_AUDIO_LEVEL_CHANGED change conference event (for local tracks).
+ * @param {TraceablePeerConnection|null} tpc
  * @param audioLevel the audio level
  */
-JitsiConference.prototype._fireAudioLevelChangeEvent = function(audioLevel) {
-    this.eventEmitter.emit(
-        JitsiConferenceEvents.TRACK_AUDIO_LEVEL_CHANGED,
-        this.myUserId(), audioLevel);
+JitsiConference.prototype._fireAudioLevelChangeEvent
+= function(tpc, audioLevel) {
+    const activeTpc = this.getActivePeerConnection();
+
+    // There can be no TraceablePeerConnection if audio levels do not come from
+    // a peerconnection (see LocalStatsCollector.js)
+    if (tpc === null || activeTpc === tpc) {
```

Comment:
It means that if audio levels are emitted by LocalStatsCollector.js there will be no TPC instance, because it does not need one to measure them (web audio API is used). I'll try to describe it better.